src/_scss/pages/search/filters/awardAmount/_specificAwardAmount.scss

```diff
@@ -26,6 +28,10 @@
             }
         }
 
+        .award-amount-submit {
+            @include singleSubmit;
```

Comment:
<img width="338" alt="specificawardamount" src="https://user-images.githubusercontent.com/7108785/33854914-57f0a8d8-de91-11e7-97d8-f2dabbc5a844.png">
Icon goes off-center between mobile and 1342 px